jbpm-wb-process-runtime/jbpm-wb-process-runtime-backend/src/main/java/org/jbpm/workbench/pr/backend/server/RemoteProcessRuntimeDataServiceImpl.java

```diff
@@ -72,6 +73,17 @@ public ProcessInstanceSummary getProcessInstance(ProcessInstanceKey processInsta
         return new ProcessInstanceSummaryMapper(processInstanceKey.getServerTemplateId()).apply(processInstance);
     }
 
+    private List<ProcessInstanceSummary> getProcessInstancesByParentId(ProcessInstanceKey processInstanceKey, String containerId){
+        if (processInstanceKey == null || processInstanceKey.isValid() == false) {
+            return null;
+        }
+
+        ProcessServicesClient processServicesClient = getClient(processInstanceKey.getServerTemplateId(), ProcessServicesClient.class);
+        List<ProcessInstance> processInstances = processServicesClient.findProcessInstancesByParent(containerId, processInstanceKey.getProcessInstanceId(), Arrays.asList(1, 2, 3), 0, Integer.MAX_VALUE);
```

Comment:
Put the statuses by the constants, are those:
    ProcessInstance.STATE_ACTIVE,ProcessInstance.STATE_COMPLETED,ProcessInstance.STATE_ABORTED
?!

Reply:
the ProcessInstance different to org.kie.api.runtime.proces.ProcessInstance.